dashboard/app/models/learning_goal_ai_evaluaton.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: learning_goal_ai_evaluatons
+#
+#  id               :bigint           not null, primary key
+#  user_id          :integer
+#  learning_goal_id :integer
+#  prompt_version   :datetime
+#  userstanding     :integer
+#  context          :text(65535)
```

Comment:
just curious, do we think this will be JSON?

Reply:
@wilkie ?